### PR TITLE
add subversion v1.6.23 as build dep for Python 2.5.6, add easyconfigs for subversion + deps

### PR DESCRIPTION
**Pull request by boegel**
for anyone reviewing this (@fgeorgatos?), just focus on one toolchain, the other easyconfigs are identical except for the `toolchain` part

### CODE REVIEW

**Comment by fgeorgatos** [comment]
let's just spare trying to convince us on the merits of git over svn ;-)

can we merge latest develop over this branch to make it ready for review?

IMHO, naming should rather be Subversion (1st letter capital) and by the timing of merging this v1.9.0 shuold be available, too.

**Comment by wpoely86** [comment]
Is this still relevant with #1970 ?

**Comment by boegel** [comment]
@wpoely86: I think it is; this is (purposely) a very old version of Subversion, which I looked into for a very specific reason (which now already has another workaround though...)

This PR needs a bit of work before it is ready, however...

**Comment by boegel** [comment]
@wpoely86: please rereview? I've trimmed things down a bit, since this svn version is no longer needed as a build dep for Python 2.5.6

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 0 out of 0 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/32de18a8eb9565ecfe68 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/037dd442b46898992fc6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 4 out of 5 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e62f404889a06cacafce for a full test report.

**Comment by wpoely86** [comment]
lgtm

**Comment by wpoely86** [comment]
@boegel the download failed. Is it really usefull to add this old version? With #1970 and `--try` we can generate this too?

**Comment by boegel** [comment]
@wpoely86: no, the deps are different

Is there an alternate download location for the old source tarball? Can you check that?

**Comment by wpoely86** [comment]
okay, but is there any reason anybody would want this old versions? svn is backwards compatible as far as I know and performance does not really come into play here.

**Comment by boegel** [comment]
There are reason I looked into this, see https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/Python/Python-2.5.6_svnversion-cmd.patch

**Comment by boegel** [comment]
Would be a shame to let the effort I spent into this go to waste.

File: easybuild/easyconfigs/s/Subversion/Subversion-1.6.23-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+name = 'Subversion'
+version = '1.6.23'
+
+homepage = 'http://subversion.apache.org/'
+description = " Subversion is an open source version control system."
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://archive.apache.org/dist/subversion/']
+sources = [SOURCE_TAR_GZ]
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/Subversion/Subversion-1.6.23-goolf-1.4.10.eb]: `SOURCELOWER_TAR_GZ` will do the trick. Even better:
`SOURCELOWER_TAR_BZ2`.

Comment (boegel) [inline easybuild/easyconfigs/s/Subversion/Subversion-1.6.23-goolf-1.4.10.eb]: ah, crap, missed that when renaming to `Subversion`, thanks

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c86bdadd2b4ac57d5406 for a full test report.

**Comment by fgeorgatos** [comment]
visual review ok, lgtm

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2098.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2747ca9815ce7b15b747 for a full test report.

**Comment by boegel** [comment]
This should have been merged ages ago... better late than never! ^_^

**Pull request merged**